### DISCUSSION TITLE
What is your favorite song?

### DISCUSSION THREAD

**Discussion by mattun-tomoki** [NAIST-SE/AttractingContributors#168] [General]
https://github.com/mattun-tomoki/AttractingContributors

**Comment by tktkbohshi** [comment]
Added!
plz help me...
https://github.com/tktkbohshi/AttractingContributors-favorite-niconico-douga

**Comment by t-matsumoto4758** [comment]
Added!
please check and help me!
https://github.com/t-matsumoto4758/AttractingContributors-YourHometown

**Comment by xincanfeng** [comment]
added.
can you help by telling me your favirate color?
https://github.com/FengXincan/AttractingContributors-color
